.github/index.html

```diff
@@ -0,0 +1,129 @@
+            </button>
+          </div>  
+          <img class="panda-pic" src="img/panda1.png" alt="fashionist_panda">
+        </div>
+      </div>
+    </section>
+    <section class="white-section">
+      <div class="all">
+        <div class="nemonemo">
+          <img class="main-pic" src="img/hukuHeart.png" alt="panda_look_clothes">
+          <div class="contents">
+            <span class="headPoint">
+              Hot item
+            </span>
+            <h2 class="title">
+              인기 상품을<br>
+              확인해 보세요
+            </h2>
+            <p class="content">가장 HOT한 중고거래 물품을 판다 마켓에서 확인해 보세요</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="white-section">
+      <div class="all">
+        <div class="nemonemo">
+          <div class="contents">
+            <span class="headPoint">
+              Search
+            </span>
+            <h2 class="title">
+              구매를 원하는<br>
+              상품을 검색하세요
+            </h2>
+            <p class="content">구매하고 싶은 물품을 검색해서 쉽게 찾아보세요</p>
+          </div>
+          <img class="main-pic" src="img/zoom.png" alt="lens-pic">
+        </div>   
+      </div>
+    </section>
+    <section class="white-section">
+      <div class="all">
+        <div class="nemonemo">
+          <img class="main-pic" src="img/sq.png" alt="sellWishProduct">
+          <div class="contents">
+            <span class="headPoint">
+              Register
+            </span>
+            <h2 class="title">
+              판매를 원하는<br>
+              상품을 등록하세요
+            </h2>
+            <p class="content">어떤 물건이든 판매하고 싶은 상품을 쉽게 등록하세요</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="blue-section">
+      <div class="all">
+        <div class="Nemo">
+          <div class="miniNemo">
+            <h2 class="title">
+              믿을 수 있는<br>
+              판다마켓 중고 거래
+            </h2>
+          </div>
+          <img class="greeting-panda-pic" src="img/pandaLa.png" alt="greeting_panda">
+        </div>
+      </div>
+    </section>
+    <footer>
+      <div class="all">
+        <div class="foot">
+          <div class="codeit-2024">
+           @codeit-2024
+         </div>
+         <div class="privacyPolicy-FAQ">
+            <a class="link-line" href="/privacy">Privacy Policy</a>
+            <a class="link-line" href="/faq">FAQ</a>
+         </div>
+         <div class="sns-link">
+            <a href="https://www.facebook.com" target="blank">
+             <img class="link-logo" src="img/Facebook.png" alt="Facebook">
+            </a>
+            <a href="https://www.twitter.com" target="blank">
+              <img class="link-logo" src="img/twiter.png" alt="twitter">
+            </a>
+            <a href="https://www.youtube.com" target="blank">
+             <img class="link-logo" src="img/youtube.png" alt="youtube">
+            </a>
+            <a href="https://www.instagram.com" target="blank">
```

Comment:
❗️ 수정요청
a 태그의 target 속성에는 예약어가 들어가야합니다~
mdn에서 입력 가능한 예약어와 동작을 확인할 수 있습니다.
새창으로 열리게 하고 싶으시면 _blank로 해주셔야 합니다~

[a tag](https://developer.mozilla.org/ko/docs/Web/HTML/Element/a#target)

```suggestion
            <a href="https://www.instagram.com" target="_blank">
```